.github/workflows/full_stack_tests.yml

```diff
@@ -77,10 +77,22 @@ jobs:
     runs-on: ubuntu-22.04
     if: ${{ fromJSON(needs.check_test_suites_to_run.outputs.TEST_SUITES_TO_RUN).e2e.count > 0 }}
     strategy:
+      fail-fast: false
       matrix:
         suite: ${{ fromJSON(needs.check_test_suites_to_run.outputs.TEST_SUITES_TO_RUN).e2e.suites }}
     name: E2E (${{ matrix.suite.name }})
     steps:
+      - name: Check for broken suites
+        id: check_skip
+        run: |
+          broken_suites=("contributorDashboard" "contributorAdminDashboard" "explorationTranslationTab" "fileUploadFeatures")
```

Comment:
@jayam04 Where is the issue that tracks all these? There should be a TODO above this line that points to that issue and says something like "Remove each of the broken suites as they get fixed."

Ditto below.

Reply:
Ah, here it is: https://github.com/oppia/oppia/issues/22448. The comment above still applies.